### PR TITLE
feat: verify connect command

### PR DESCRIPTION
**Pull request by filipslezaklab**
Verify that made connection is alive via handshake check.

fix: settings styling
fix: location / tunnels view display
feat: extend client settings
chore: port application configuration solution from encrypt db PR
chore: refactor dead connection dropped event

### CODE REVIEW

File: src-tauri/src/database/models/location_stats.rs [hunk 1]
```diff
@@ -178,4 +178,31 @@ impl LocationStats<Id> {
...
+
+    pub async fn latest_by_location_id<'e, E>(
+        executor: E,
+        location_id: Id,
+    ) -> Result<Option<Self>, Error>
+    where
+        E: SqliteExecutor<'e>,
+    {
+        let res = query_as!(
+            LocationStats::<i64>,
```
Comment (moubctez) [inline src-tauri/src/database/models/location_stats.rs:190]:
```suggestion
            LocationStats::<Id>,
```

File: src-tauri/src/database/models/tunnel.rs [hunk 2]
```diff
@@ -357,6 +357,34 @@ impl TunnelStats<Id> {
...
+
+    pub async fn latest_by_tunnel_id<'e, E>(
+        executor: E,
+        tunnel_id: Id,
+    ) -> Result<Option<Self>, Error>
+    where
+        E: SqliteExecutor<'e>,
+    {
+        let res = query_as!(
+            TunnelStats::<i64>,
```
Comment (moubctez) [inline src-tauri/src/database/models/tunnel.rs:369]:
```suggestion
            TunnelStats::<Id>,
```

File: src-tauri/src/utils.rs [hunk 3]
```diff
@@ -1208,3 +1210,182 @@ pub async fn sync_connections(apphandle: &AppHandle) -> Result<(), Error> {
...
+                .lock()
+                .await
+                .clone()
+                .iter()
+                .find(|x| {
+                    x.location_id == *&location.id && x.connection_type == ConnectionType::Location
+                }) {
+                Some(active_connection) => {
+                    debug!("Verifying connection to location {location}");
+                    trace!("Verifying connection {:?}", active_connection);
```
Comment (moubctez) [inline src-tauri/src/utils.rs:1258]:
```suggestion
                    trace!("Verifying connection {active_connection:?});
```

File: src-tauri/src/utils.rs [hunk 4]
```diff
@@ -1208,3 +1210,182 @@ pub async fn sync_connections(apphandle: &AppHandle) -> Result<(), Error> {
...
+
+    match connection {
+        ConnectionToVerify::Location(location) => {
+            match state
+                .active_connections
+                .lock()
+                .await
+                .clone()
+                .iter()
+                .find(|x| {
```
Comment (moubctez) [inline src-tauri/src/utils.rs:1253]: Powinno być łatwiej z `.find(|&x|`

File: index.html [hunk 5]
```diff
@@ -1,14 +1,18 @@
...
+</head>
+
+<body>
+  <div id="root"></div>
+  <div id="modals-root"></div>
+  <div id="toasts-root"></div>
+  <script type="module" src="/src/main.tsx"></script>
+</body>
+
+</html>
```
Comment (t-aleksander) [inline index.html:18]:
```suggestion
</html>

```

**Pull request merged**